src/inventory/gpu.go

```diff
@@ -0,0 +1,91 @@
+package inventory
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/openshift/assisted-installer-agent/src/util"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+type lshw struct {
+	Product string `json:"product,omitempty"`
+	Vendor  string `json:"vendor,omitempty"`
+	Clock   int64  `json:"clock,omitempty"`
+	BusInfo string `json:"businfo,omitempty"`
+}
+
+func GetGPUs(dependencies util.IDependencies) []*models.Gpu {
+	gpus := make([]*models.Gpu, 0)
+	o, e, exitCode := dependencies.Execute("lshw", "-C", "display", "-json")
+	if exitCode != 0 {
+		logrus.Warnf("Error running lshw: %s", e)
+		return gpus
+	}
+	var l []lshw
+	o = sanitizeLshwJsonOutput(o)
+	err := json.Unmarshal([]byte(o), &l)
+	if err != nil {
+		logrus.Warnf("Error unmarshalling lshw: %s", err.Error())
```

Comment:
Add the input. Otherwise, we wouldn't be able to know what wasn't able to be unmarshalled.
```suggestion
		logrus.WithError(err).Warnf("Error unmarshalling lshw: %s", o)
```